### PR TITLE
docs: add kameleoon rum integration (ECOINT-10)

### PR DESCRIPTION
**Pull request by Slava-Inyu**
### What does this PR do?

Adds Kameleoon integration with DataDog RUM

### Motivation

Related to this [documentation PR](https://github.com/DataDog/documentation/pull/24903)

### Review checklist

- [ ] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [ ] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)
- [ ] If this PR includes a log pipeline, please add a description describing the remappers and processors.

### CODE REVIEW

**Review by buraizu — CHANGES_REQUESTED** [review]
Thanks, just requesting some minor updates

File: kameleoon/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+```javascript
+client.onEvent(EventType.Evaluation, ({ featureKey, variation }) => {
+  datadogRum.addFeatureFlagEvaluation(featureKey, variation.key);
+});
+```
+
+## Support
+
+For more information see [Kameleoon SDK documentation][2] or join [Kameleoon Slack community][3] for support on Kameeloon Datadog integration.
```
[the remark this answers is not in the document]
Comment (Slava-Inyu) [reply kameleoon/README.md]: Thanks for suggestions
Updated

File: kameleoon/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+Integrate with Datadog RUM to monitor feature deployments and releases with real-time performance data, helping you understand the direct impact of specific features on user behavior and application metrics.
+
+## Setup
+
+Feature flag tracking is available in the RUM Browser SDK. For detailed set up instructions, visit [Getting started with feature flag data in RUM][1].
+
+1. Update your Browser RUM SDK version to 4.25.0 or above.
+2. Initialize the RUM SDK and configure the `enableExperimentalFeatures` initialization parameter with `["feature_flags"]`.
+3. Initialize [Kameleoon's SDK][2] and define `onEvent` handler to watch `Evaluation` events.
```
[the remark this answers is not in the document]
Comment (Slava-Inyu) [reply kameleoon/README.md]: Updated

**Review by emarsha94 — CHANGES_REQUESTED** [review]
Could you please add 3 images of the integration that will display on your tile, as well as a logo in SVG format

**Comment by Slava-Inyu** [comment]
@emarsha94 , we added 2 if it's not a problem, couldn't find screenshot info diverse enough for 3

**Review by buraizu — APPROVED** [review]

**Comment by Slava-Inyu** [comment]
> Could you please add 3 images of the integration that will display on your tile, as well as a logo in SVG format

@emarsha94 , I did an update with 2 screenshots if it's not a problem, anything else that needs to be updated?

File: kameleoon/CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# CHANGELOG - Kameleoon
+
+## 1.0.0 / 2024-08-27
```
Comment (emarsha94) [inline kameleoon/CHANGELOG.md:3]:
```suggestion
## 1.0.0 / 2024-09-18
```

**Review by emarsha94 — APPROVED** [review]

**Review by steveny91 — APPROVED** [review]

**Review by james-eichelbaum — APPROVED** [review]

**Pull request merged**